doc/level1.rst

```diff
@@ -934,46 +934,243 @@ Example Setup Message
 
 ::
 
-   To: me@mydomain.com
-   From: me@mydomain.com
-   Autocrypt-Setup-Message: v1
-   Content-type: multipart/mixed; boundary="==break1=="
-
-   --==break1==
-   Content-Type: text/plain
+    Date: Sun, 05 Nov 2017 08:44:38 GMT
+    To: alice@autocrypt.example
+    From: alice@autocrypt.example
+    Autocrypt-Setup-Message: v1
+    Subject: Autocrypt Setup Message
+    Content-type: multipart/mixed; boundary="Y6fyGi9SoGeH8WwRaEdC6bbBcYOedDzrQ"
+
+    --Y6fyGi9SoGeH8WwRaEdC6bbBcYOedDzrQ
+    Content-Type: text/plain
 
-   This is the Autocrypt setup message.
+    This message contains all information to transfer your Autocrypt settings along with your secret key securely from your original device.
 
-   --==break1==
-   Content-Type: application/autocrypt-key-backup
-   Content-Disposition: attachment
+    To set up your new device for Autocrypt, please follow the instuctions that should be presented by your new device.
 
-   Possibly a descriptive text, informing users about this file's
-   contents.
-   -----BEGIN PGP MESSAGE-----
-   Passphrase-Format: numeric9x4
-   Passphrase-Begin: 31
+    You can keep this message and use it as a backup for your secret key. If you want to do this, you should write down the password and store it securely.
+    --Y6fyGi9SoGeH8WwRaEdC6bbBcYOedDzrQ
+    Content-Type: application/autocrypt-setup
+    Content-Disposition: attachment; filename="autocrypt-key-backup.html"
```

Comment:
why is filename="autocrypt-key-backup.html" ?  shouldn't this be "autocrypt-setup-message.html" for consistency in interface?

Reply:
I would omit the html trick from this example. It's not obvious how this works or why it is useful, and in the spec we only mention it in reference to a thought in an external document.